### ISSUE TITLE
Curly brackets break remote commands on Linux hosts since 5.0

### ISSUE THREAD

**Issue by mazzella-c** [rundeck/rundeck#8773]
**Describe the bug**
Curly brackets are not read anymore since 5.0 release on Linux hosts.

**My Rundeck detail**

* Rundeck version: 5.0
* install type: war
* OS Name/version: RHEL 7.9
* DB Type/version: Postgres

**To Reproduce**
1. Set test_value variable to something on a Linux host
2. Launch a simple test by executing echo _${test_value}_ on Rundeck with a dedicated job using remote command or directly with a command on the node
3. Result is empty with _${test_value}_ while OK with _$test_value_
4. The same test ends OK on 4.17.3

**Expected behavior**

Correct display of command using curly brackets

**Comment by MegaDrive68k** [comment]
Hi @mazzella-c 

Could you share a job definition example to take a look at?

Thanks!

**Comment by MegaDrive68k** [comment]
Never mind, the issue is confirmed.

Job def:

```yaml
- defaultTab: nodes
  description: ''
  executionEnabled: true
  id: 7d0b9606-4e9d-4b53-b9c8-e4d20b0ec26e
  loglevel: INFO
  name: HelloWorld
  nodeFilterEditable: false
  plugins:
    ExecutionLifecycle: {}
  scheduleEnabled: true
  sequence:
    commands:
    - exec: test_value=15; echo ${test_value}
    keepgoing: false
    strategy: node-first
  uuid: 7d0b9606-4e9d-4b53-b9c8-e4d20b0ec26e

```

4.17.0:

![Screenshot_2024-01-09_10-34-41](https://github.com/rundeck/rundeck/assets/51376003/fc2ac76d-e154-405b-96f7-511948d14376)


5.0.0:

![Screenshot_2024-01-09_10-33-20](https://github.com/rundeck/rundeck/assets/51376003/bdc10d1c-3aa9-4c21-8285-9df63bf9135c)

Thanks for your feedback @mazzella-c!

**Comment by mshedsilegx** [comment]
can we prioritize resolution of this ? The regression behavior of 5.0 is causing all our jobs to fail, as they use ${VARIABLE}, not $VARIABLE

**Comment by MegaDrive68k** [comment]
The issue is fixed on 5.0.1, folks. Please upgrade.

![Screenshot_2024-01-16_17-21-07](https://github.com/rundeck/rundeck/assets/51376003/e06d8960-6ab5-4f81-acea-1fe04da01457)
